docs/cugraph/source/api_docs/structure.rst

```diff
@@ -1,31 +1,40 @@
-=========
-Structure
-=========
-.. currentmodule:: cugraph.structure.graph_classes
+=============
+Graph Classes
+=============
+.. currentmodule:: cugraph
 
-Constructor
------------
+Constructors
+------------
 .. autosummary::
    :toctree: api/
 
    Graph
+   MultiGraph
+   BiPartiteGraph
 
 
-Methods
--------
+Adding Data
```

Comment:
This is kind of a nit pick so feel free to ignore: this list also includes `clear()`, so maybe the header is "Adding and Removing Data"? This could also be a good placeholder for future filtering and dynamic graph APIs.